hack/e2e.go

```diff
@@ -46,6 +46,7 @@ func parse(args []string) (flags, error) {
 	fs := flag.NewFlagSet(args[0], flag.ContinueOnError)
 	get := fs.Bool("get", getDefault, "go get -u kubetest if old or not installed")
 	old := fs.Duration("old", oldDefault, "Consider kubetest old if it exceeds this")
+	verbose := fs.Bool("v", true, "this flag is translated to kubetest's --verbose-commands")
```

Comment:
> kubetest now imports client-go, which imports glog, which registers an incompatible -v level flag

doesn't registering `v` into the local flagset mask the global one? and if `--v=5` is passed, this will fail with a parse error, right?

Reply:
we're shelling out to kubetest, not importing it. The goal is *explicitly* to mask it because people use bare `-v` (no args) already which doesn't match what kubetest now has. the old kubetest flag was a bool.